### PR TITLE
[Polygon Edge] Add troubleshooting section

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

File: docs/edge/Contracts.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  
+---
+
+# Troubleshooting
+
+## `method=eth_call err="invalid signature"` error
+
+When you are using a wallet to make a transaction with Polygon Edge, please make sure that in your wallet's local network setup:
+<ul>
+<li>The `chainID` is the right one. The default `chainID` for Edge is `100`, but it can be customized by using the genesis flag `–chain-id`:
```
Comment (zivkovicmilos) [inline docs/edge/Contracts.md:19]: Flags are specified with 2 dashes, not one: `--chain-id`

File: docs/edge/Contracts.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+</ul>
+
+## Send button on wallet is greyed out
+Firstly, please check if this is not a wallet related issue. If the issue still persists, and if you can also see error messages in your logs, make sure that you are using the correct path for the `--ibft-validators-prefix-path` flag while creating genesis.json.
+
+For example:
+<ul>
+<li> If you use the command `polygon-edge secrets init --data-dir validator-x` to generate the keys for the validators, the right way to use the flag would be `--ibft-validators-prefix-path validator-`; x-validator’s number </li>
+<li>If you use the command `polygon-edge secrets init --data-dir test-chain-x` to generate the keys for the validators, the right way to use the flag would be `--ibft-validators-prefix-path test-chain-`; x-validator’s number </li>
+</ul>
```
Comment (zivkovicmilos) [inline docs/edge/Contracts.md:33]: What does this have to do with the Send button being grayed out?

cc @Aleksao998

File: docs/edge/Contracts.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## How to get a WebSocket URL
+
+By default, when you run the Polygon Edge, it generates a WebSocket URL based on the chain location.
```
Comment (zivkovicmilos) [inline docs/edge/Contracts.md:37]: It doesn't generate anything, it _exposes_ a websocket _endpoint_

File: docs/edge/Contracts.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+The URL scheme `wss://` is used for HTTPS links, and `ws://` for HTTP.
+
+Localhost WebSocket URL:
+````bash
+ws://localhost:10002/ws
```
Comment (zivkovicmilos) [inline docs/edge/Contracts.md:42]: I would replace this to be in the form of:

`ws://<JSON-RPC URL>:<PORT>/ws`

File: docs/edge/Contracts.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+Edgenet WebSocket URL:
+````bash
+wss://rpc-edgenet.polygon.technology/ws
+````
+
+## `insufficient funds` error when trying to deploy a contract
+
+If you get this error, please make sure you have a premined balance on the desired address, and the address used is the correct one.
+To set the premined balance, you can use the genesis flag `genesis [--premine ADDRESS:VALUE]` while generating the genesis file.
+Example of using this flag: `genesis --premine 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862:1000000000000000000000`. This premines 1000000000000000000000 WEI to 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862.
```
Comment (zivkovicmilos) [inline docs/edge/Contracts.md:55]: You don't need to premine to deploy a contract - you need to have enough funds.

File: docs/edge/Contracts.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+ 
+ 
+## ERC20 tokens not released while using Chainbridge
+
+If you try to transfer ERC20 tokens between Polygon POS and a local Edge network, and your  ERC20 tokens are deposited, also proposal is executed at relayer, but the tokens are not released in your Edge network, please make sure the ERC20 Handler in Polygon Edge chain has enough tokens to release.
```
Comment (zivkovicmilos) [inline docs/edge/Contracts.md:60]: Nitpick, double space between `your` and `ERC20`

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯 

Left a few minor comments

File: docs/edge/Contracts.md (hunk 1 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: fixed d1c09b246bf66a548ddfd33a2d194fbc2cbfd084

File: docs/edge/Contracts.md (hunk 2 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: @zivkovicmilos The info was taken from this [issue](https://github.com/0xPolygon/polygon-edge/issues/443) . Let me know what changes you'd suggest.

File: docs/edge/Contracts.md (hunk 3 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: fixed e8ffd4781eafd7df36d944dd1c50c28c9f4b973a

File: docs/edge/Contracts.md (hunk 5 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: fixed 5ec12e2c068eb52b815be95d33539748ffe83323

File: docs/edge/Contracts.md (hunk 6 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: fixed dc28ce6d52c95de5165544f46ec9742336c21b15

File: docs/edge/Contracts.md (hunk 4 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: fixed e86cf4a424adbbdc8c71c5d2810fe132fdcf2479

File: docs/edge/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  - troubleshooting
+  
+---
+
+# Troubleshooting
+
+## `method=eth_call err="invalid signature"` error
+
+When you are using a wallet to make a transaction with Polygon Edge, please make sure that in your wallet's local network setup:
+<ul>
```
Comment (Aleksao998) [inline docs/edge/troubleshooting.md:18]: My advice is not to use <ul> and <li> tags. when you put text inside these tags you wouldn't have option to have specific formating that md files provides 

example: `chainID` would really look like **`chainID`** it wouldn't format it in any way

My suggestion use order list: 

When you are using a wallet to make a transaction with Polygon Edge, please make sure that in your wallet's local network setup:

1. The `chainID` is the right one. The default `chainID` for Edge is `100`, but it can be customized by using the genesis flag `--chain-id`.

````bash
genesis [--chain-id CHAIN_ID]
```` 

2. Make sure that, on the “RPC URL”, field you use the JSON RPC port of the node you are connecting to

File: docs/edge/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+</li>
+<li>Make sure that, on the “RPC URL”, field you use the JSON RPC port of the node you are connecting to</li>
+</ul>
+
+## Send button on wallet is greyed out
+Firstly, please check if this is not a wallet related issue. If the issue still persists, and if you can also see error messages in your logs, make sure that you are using the correct path for the `--ibft-validators-prefix-path` flag while creating genesis.json.
+
+For example:
+<ul>
+<li> If you use the command `polygon-edge secrets init --data-dir validator-x` to generate the keys for the validators, the right way to use the flag would be `--ibft-validators-prefix-path validator-`; x-validator’s number </li>
```
Comment (Aleksao998) [inline docs/edge/troubleshooting.md:33]: same as prev comment

File: docs/edge/troubleshooting.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+Please note that the port number depends on the chosen JSON-RPC port for the node.
+
+Edgenet WebSocket URL:
+````bash
+wss://rpc-edgenet.polygon.technology/ws
+````
+
+## `insufficient funds` error when trying to deploy a contract
+
+If you get this error, please make sure you have a enough funds on the desired address, and the address used is the correct one.
```
Comment (Aleksao998) [inline docs/edge/troubleshooting.md:55]: This whole section needs to be formatted much better. It's missing new lines and for all bigger commands i would advice 

````bash
genesis --premine 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862:1000000000000000000000
````

rather then 
`genesis --premine 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862:1000000000000000000000`

File: docs/edge/troubleshooting.md (hunk 7 above)
Comment (laviniat1996) [reply docs/edge/troubleshooting.md]: @Aleksao998 I've made one big commit for all your comments aebab5cbebe491760aeeebdd0345ff12fde65c25

File: docs/edge/troubleshooting.md (hunk 8 above)
Comment (laviniat1996) [reply docs/edge/troubleshooting.md]: @Aleksao998 I've made one big commit for all your comments aebab5cbebe491760aeeebdd0345ff12fde65c25

File: docs/edge/troubleshooting.md (hunk 9 above)
Comment (laviniat1996) [reply docs/edge/troubleshooting.md]: @Aleksao998 I've made one big commit for all your comments aebab5cbebe491760aeeebdd0345ff12fde65c25

File: docs/edge/Contracts.md (hunk 2 above)
Comment (zivkovicmilos) [reply docs/edge/Contracts.md]: This has nothing to do with txn sending in Metamask - I suggest we drop this section completely

File: docs/edge/Contracts.md (hunk 5 above)
Comment (zivkovicmilos) [reply docs/edge/Contracts.md]: Typo there, should be `make sure you have enough funds`

File: docs/edge/Contracts.md (hunk 2 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: fixed 86335c3fdff85d4defc4ace029df5c8d5f1d8aed

File: docs/edge/Contracts.md (hunk 5 above)
Comment (laviniat1996) [reply docs/edge/Contracts.md]: fixed 86335c3fdff85d4defc4ace029df5c8d5f1d8aed

**Review by Aleksao998 — APPROVED** [review]
LGTM
